### PR TITLE
Disabling GPU acceleration by setting env variable

### LINKED ISSUE #222: Garbled graphics with external 4K monitor
**Describe the bug**
The graphics is all garbled on my MacBook when connected to an external monitor.
If I open the app without the monitor connected it looks OK, even if I reconnect the monitor afterwards.

**To Reproduce**
When first opening the app and the MacBook is connected to an external 4K monitor via USB-C

**Screenshots**
Screenshot 2020-04-06 at 10 11 43

**Environment (please complete the following information):**
- Lens Version: 3.2.0-rc.1
- OS: MacOS 10.15.4

### LINKED ISSUE #462: Option to disable GPU acceleration
**What would you like to be added**:

Option to disable GPU acceleration.

**Why is this needed**:

MacOS seems to have issues with chrome/chromium: https://www.forbes.com/sites/gordonkelly/2020/06/11/apple-macos-macbook-pro-google-chrome-display-problem/amp/

### LINKED ISSUE #562: Fuzzy display on Apple Pro Display XDR
**Describe the bug**
Opening the application shows a fuzzy display. When a cluster is selected, everything works as expected.

**To Reproduce**
1. Open the application

**Expected behavior**
Initial screen should show correctly.

**Screenshots**
image

**Environment (please complete the following information):**
- Lens Version: Version: 3.5.1
- OS: Catalina
- Installation method (e.g. snap or AppImage in Linux): Git release

### PR DESCRIPTION
**Pull request by aleksfront**
It disables GPU acceleration if `LENS_DISABLE_GPU` env var is provided. Also, added a bit of description in `/troubleshooting` section.

This PR will replace this one https://github.com/lensapp/lens/pull/1123 because #1123 won't allow to activate non-GPU mode in case of broken window rendering.

Fixes #462
Fixes #222
Fixes #562

Signed-off-by: Alex Andreev <alex.andreev.email@gmail.com>

### CODE REVIEW

**Comment by aleksfront** [comment]
Result of `app.getGPUFeatureStatus()` (https://www.electronjs.org/docs/all#appgetgpufeaturestatus) with flag provided:
```
{
  '2d_canvas': 'unavailable_software',
  flash_3d: 'disabled_software',
  flash_stage3d: 'disabled_software',
  flash_stage3d_baseline: 'disabled_software',
  gpu_compositing: 'disabled_software',
  metal: 'disabled_off',
  multiple_raster_threads: 'enabled_on',
  oop_rasterization: 'disabled_off',
  opengl: 'disabled_off',
  protected_video_decode: 'disabled_off',
  rasterization: 'disabled_software',
  skia_renderer: 'disabled_off_ok',
  video_decode: 'disabled_software',
  webgl: 'unavailable_software',
  webgl2: 'unavailable_software'
}
```

File: troubleshooting/fuzzy-window-rendering.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Fuzzy window rendering on external 4K display
```
Comment (jakolehm) [inline troubleshooting/fuzzy-window-rendering.md]: I think this should be in docs.

Comment (aleksfront) [reply troubleshooting/fuzzy-window-rendering.md]: Do you think FAQ section is a good choice?

Comment (jakolehm) [reply troubleshooting/fuzzy-window-rendering.md]: > Do you think FAQ section is a good choice?

Yes.

**Comment by aleksfront** [comment]
FAQ bug description:
fuzzy rendering

**Review by jakolehm — DISMISSED** [review]
LGTM

**Review by ixrock — DISMISSED** [review]

**Review by jakolehm — APPROVED** [review]

**Review by ixrock — APPROVED** [review]

**Pull request merged**